app/workers/database.php

```diff
@@ -155,10 +164,18 @@ protected function getInternalDB($projectId): Database
     protected function getExternalDB($projectId): Database
     {
         global $register;
-        
-        $cache = new Cache(new RedisCache($register->get('cache')));
-        $dbForExternal = new Database(new MariaDB($register->get('db')), $cache);
-        $dbForExternal->setNamespace('project_'.$projectId.'_external'); // Main DB
+
+        $dbForExternal = null;
+
+        go(function() use ($register, $projectId, &$dbForExternal) {
```

Comment:
@TorstenDittmann @kodumbeats are we sure this update has no side effects? Would it be safer to apply the `go()` function only on the db setup process instead of on any new connection?